recipes/libattr/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+add_executable(${PROJECT_NAME} example.c)
+target_link_libraries(${PROJECT_NAME} attr)
```

This fixes the c3i error.
By passing `TARGETS` to `conan_basic_setup`, it will create `CONAN_PKG::` targets and not do `include_directories/add_definitions`.


```suggestion
conan_basic_setup()

add_executable(${PROJECT_NAME} example.c)
target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
```

An alternative to the above suggestion would be to keep the `TARGETS` and link to `CONAN_PKG::libattr` instead.